.travis.yml

```diff
@@ -74,3 +75,8 @@ jobs:
       env: PLATFORM=imx8x
     - <<: *qemuboottest
       env: PLATFORM=imx8m
+    - stage: testbench
+      script:
+        - ./scripts/docker-run.sh ./scripts/build-tools.sh -t &> /dev/null
```

Comment:
Please never ever discard all output like this, especially not in CI where the exact configuration can be difficult to reproduce locally. If the script is too verbose then its verbosity can be adjusted but discarding everything is throwing the baby with the bathwater.

Fixed in #3364